packages/fxa-payments-server/src/components/PaymentMethodHeader/index.test.tsx

```diff
@@ -0,0 +1,118 @@
+import React from 'react';
+import '@testing-library/jest-dom/extend-expect';
+import TestRenderer from 'react-test-renderer';
+import { cleanup, fireEvent, render } from '@testing-library/react';
+import { PaymentMethodHeader } from '.';
+import {
+  getLocalizedMessage,
+  MOCK_PLANS,
+  setupFluentLocalizationTest,
+} from '../../lib/test-utils';
+import { act } from 'react-dom/test-utils';
+import { getLocalizedCurrency } from '../../lib/formats';
+import waitForExpect from 'wait-for-expect';
+
+jest.mock('../../lib/sentry');
+
+describe('components/PaymentMethodHeader', () => {
+  afterEach(() => {
+    jest.clearAllMocks();
+    return cleanup();
+  });
+
+  it('render header without prefix', async () => {
+    const plan = MOCK_PLANS[0];
+    const props = { plan, onClick: () => {} };
+    const { queryByTestId } = render(<PaymentMethodHeader {...props} />);
+
+    await waitForExpect(() => {
+      expect(queryByTestId('header')).toBeInTheDocument();
+      expect(queryByTestId('header-prefix')).not.toBeInTheDocument();
+    });
```

Comment:
This is fine.  But FYI, if you wrap code that causes (re-)rendering in `act`, e.g.
```
await act(async () => render(...));
```
then you don't need to use `waitForExpect`.

Reply:
Ah cool, yup that makes sense. I removed the waitForExpect, and works as...expected (excuse the pun)